src/App.jsx

```diff
@@ -38,11 +38,11 @@ function App () {
   const [showDevtools] = useState(webAppConfig.ENABLE_REACT_QUERY_TOOLS !== undefined ? webAppConfig.ENABLE_REACT_QUERY_TOOLS : true);
 
 
-  // Inject this once for the app, for all react-query queries
+  // Inject this once for the app, then it is the default for all ReactQuery queries
   const queryClient = new QueryClient({
     defaultOptions: {
       queries: {
-        networkMode: 'always', // Send queries to the server even if the cache has the data
+        // networkMode: 'always', // <-- This is not a solution, it just covers up some problem in our code, while disabling the biggest benefit of ReactQueries. Send queries to the server even if the cache has the data
```

Comment:
For testing purposes, we need to have a reliable way to tell react-query to not cache anything. I don't think we have proven that we 100% understand all of the cases where react-query pulls data from its cache vs. directly from the API server. By being able to turn on a react-query configuration that forces only the freshest data from the API server, we can then isolate inconsistencies if there are any. Let's maintain the ability to test our theories about how react-query works in all edge cases before we assume that any caching or retrieval data bugs that we find are our fault. The react-query documentation describes a lot of conditions where it draws from the cache instead of pulling fresh data from the API server, and it will take time for us to experience all of those cases.

Reply:
networkMode: 'always' should never be in production code, maybe wrap it in conditional test logic?  ReactQuery works 100% of the time for me, and I've tested it extensively.  It would be great if you showed me a case where it doesn't work.